### PR TITLE
Update ratings stats table

### PR DESCRIPTION
**Pull request by xavier506**
### GH Issue

This is a script that update `ratings_stats` table on postgres by reading the blockchain `stats` table. The demux service is running several hundred block behind and this can be a temporary work around until we get demux optimized or switch to a service such as dfuse.io

The idea is to run a cron job against this script that keeps ratings updated, similar to the cron job that uodates the BP information fron the chain and fetches the `bp.json`.

### Steps to test

1. checkout the `update-ratings-stats` branch 
2. install dependancies `yarn` 
3.  run `node services/update-bps/update-ratings.js `

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1,215 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (danazkari) [inline yarn.lock]: @xavier506 why was this file deleted?

Comment (danazkari) [reply yarn.lock]: I'm guessing it's because the front-end was moved to `services/frontend` but just want to make sure we don't really need this anymore.

Comment (xavier506) [reply yarn.lock]: It got deleted because all package files are under the services directory , it is an orphaned file.
